README.md

```diff
@@ -84,13 +123,17 @@ Noto is also part of my [portfolio](https://alialbaali.com) projects. It showcas
 
 ## Contributions
 
-Contributions are always welcome! If you would like to contribute, please feel free to file an issue or create a PR.
+<p>
+  Contributions are always welcome! If you would like to contribute, please feel free to file an issue or create a PR.
+</p>
 
 #### Special thanks to everyone who has helped and contributed in the development.
 
 ## Translation and Localization
 
-If you would like to translate the app, please contact me via [email](mailto:ali@albaali.com).
+<p>
+  If you would like to translate the app, please contact me via [email](mailto:ali@albaali.com).
```

Comment:
Email link isn't working here.

Reply:
The emailto in the new readme appears to be fixed.